src/Api.Rest/Common/Client/RestClientBase.cs

```diff
@@ -229,6 +230,26 @@ public bool UseDefaultWebProxy
 					_HttpClientHandler.UseProxy = value;
 			}
 		}
+
+		/// <summary>
+		/// Gets or sets a value that indicates whether the certificate is checked against the certificate authority revocation list.
+		/// </summary>
+		/// <remarks>
+		/// For executing within a browser this property is ignored.
+		/// </remarks>
+		public bool CheckCertificateRevocationList

```

Comment:
Can we add builder support for the new setting?

Reply:
Yes, of course ;) Done.